worldeditor/src/editors/contentbrowser/contentbrowser.cpp

```diff
@@ -238,6 +247,24 @@ void ContentBrowser::onItemDelete() {
     }
 }
 
+void ContentBrowser::onTreeItemRename() {
```

Comment:
Seems like this part of code is very similar to onItem functions with one exception (Model is different). Probably it can be United to remove code duplication. (for example with additional functions which can accept parameters from the models)